README.md

```diff
@@ -1,5 +1,12 @@
 # BCRegistry
 
+This project requires node v14.x.x (npm v6.14.x). You can use Node Version manager (nvm) to manage several versions of node...
```

Comment:
Actually I think it needs v16 or v17... I will fix this.

Reply:
it actually is supposed to be the same as ppr: Node v15.14.0 and npm 7.7.6

Reply:
See other conversation on Node version...